upgrade/run_upgrade_test.sh

```diff
@@ -30,12 +30,12 @@ ROOT=$(dirname "$WD")
 # Set up inputs needed by /istio/istio/tests/upgrade/test_crossgrade.sh
 # These environment variables are passed by /istio/test-infra/prow/cluster/jobs istio periodic upgrade jobs
 export SOURCE_HUB=${SOURCE_HUB:-"docker.io/istio"}
-export TARGET_HUB=${TARGET_HUB:-"docker.io/istio"}
-export SOURCE_TAG=${SOURCE_TAG:-"1.3.0"}
-export TARGET_TAG=${TARGET_TAG:-"1.3.4"}
-export SOURCE_RELEASE_PATH=${SOURCE_RELEASE_PATH:-"https://storage.googleapis.com/istio-build/dev"}
+export TARGET_HUB=${TARGET_HUB:-"gcr.io/istio-testing"}
+export SOURCE_TAG=${SOURCE_TAG:-"1.7.0-rc.1"}
```

Comment:
@linsun I'm not sure how this test works, but if you are looking to use a pre-release tag, the correct hub is `gcr.io/istio-prerelease-testing`.

The `gcr.io/istio-testing` hub contains per-commit builds, and it's useful if you want to test against the latest build of a given branch. For example, to use latest image of 1.7, use the tag `1.7-dev`. For master, use the tag `1.8-dev`.

Reply:
@linsun See the difference between both:
https://console.cloud.google.com/gcr/images/istio-testing/GLOBAL/pilot?gcrImageListsize=30
and
https://console.cloud.google.com/gcr/images/istio-prerelease-testing/GLOBAL/pilot?gcrImageListsize=30

Reply:
I'm grabbing 1.7.0-rc.1 from dockerhub.  So the idea is source should be latest published on docker hub, and target be the latest master.

Reply:
so, in this terminology, source == current and target == next, right?

Reply:
source should be 1.7 (released or latest) target should be master for master branch.

for release-1.7 branch, source should be 1.6.0 and target should be 1.7 (latest)